_posts/2019-12-02-osg-user-school-2019.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "OSG User School 2019"
+date: 2019-12-02 12:00:00 -0600
+categories: School
```

Comment:
Add:
```
author: Tim Cartwright cat@cs.wisc.edu
```
It will automatically add the author to just below the article title.  Like this:
![image](https://user-images.githubusercontent.com/79268/70070574-79834300-15b9-11ea-8575-04e49bb20619.png)

Reply:
No other post except your GPU one does that…

Reply:
Indeed, I just added it.